lib/preload/ndb/preload.js

```
@@ -5,48 +5,47 @@
  */
 
 try {
-  const inspector = require('inspector');
-  const fs = require('fs');
-  let inspectorPort = 0;
-  inspector.open(inspectorPort, undefined, false);
-  let inspectorUrl = inspector.url();
-  if (process.version.startsWith('v8.')) {
-    const url = require('url');
-    const parsedInspectorUrl = url.parse(inspectorUrl);
-    parsedInspectorUrl.pathname = '/json';
-    parsedInspectorUrl.protocol = 'http';
-    inspectorUrl = url.format(parsedInspectorUrl);
-    inspectorPort = Number(parsedInspectorUrl.port);
+  let scriptName = '';
+  try {
+    scriptName = require.resolve(process.argv[1]);
+  } catch (e) {
```

nit: empty catch block, maybe add a comment?

---

done